src/modules/IscsiClientLib.rb

```diff
@@ -871,12 +861,8 @@ def setValue(name, value)
 
     # check whether two given IP addresses (including ports) are equal
     def ipEqual?(session_ip, current_ip)
-      if !session_ip || !current_ip
-        return false
-      end
-      if session_ip.empty? || current_ip.empty?
-        return false
-      end
+      return false if !session_ip || !current_ip
+      return false if session_ip.empty? || current_ip.empty?
```

Comment:
NP: we could merge these conditions if it still fits into the line length limit.